app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -48,6 +61,12 @@ def sort_key
           "#{mapping_path.to_s.length}#{name}"
         end
 
+        def destroy_operation?(xml_node)
+          xml_node.dig('metainfo', 'opType') == Sequel::Plugins::Oplog::DESTROY_OPERATION &&
+            primary? &&
+            TradeTariffBackend.handle_soft_deletes?
+        end
```

Comment:
I am wondering if `destroy_operation?` should be inside the mapper? 🤔 
... anyway, not sure yet what would be the best place for it

Reply:
Yeah, that was my initial implementation and I came into callback scoping rules. I'm now passing the mapper instance to the callbacks that are interested in it